docs/src/main/asciidoc/qute.adoc

```diff
@@ -171,6 +171,37 @@ public class ItemResource {
 <2> Make the `Item` object accessible in the template.
 <3> A static template extension method can be used to add "computed properties" to a data class. The class of the first parameter is used to match the base object and the method name is used to match the property name.
 
+== Built-in Template Extension methods
+
+Quarkus provides a set of built-in extension methods.
+
+=== Map extension methods:
+
+* `keys()` or `keySet()`: Returns a Set view of the keys contained in a map
+** `{#for key in map.keySet()}`
```

Comment:
The brackets are optional and I'd say that `map.keySet` is more readable. The same applies to all snippets.